presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/DatabaseShardManager.java

```diff
@@ -493,6 +497,448 @@ private static int batchSize(Connection connection)
         return (connection instanceof JdbcConnection) ? 1 : 1000;
     }
 
+    // TODO: Will merge these new function with old function once new feature is stable
```

Comment:
nit: Try to be specific about which function is redundant and suggest a refactor plan.